### DISCUSSION TITLE
set-output command error

### DISCUSSION THREAD

**Discussion by halftomato** [stefanzweifel/git-auto-commit-action#248] [Help] [answered]
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Comment by stefanzweifel** [comment, marked as answer]
The Action has been updated in v4.15.2 to use the new syntax to set it's output.
https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v4.15.2
